build-logic-settings/dependency-plugin/src/main/kotlin/convention-dependencies.settings.gradle.kts

```diff
@@ -53,13 +53,20 @@ dependencyResolutionManagement {
                 includeGroup("com.google.testing.platform")
             }
         }
+        /**
+         * This repo is used for kotlinx-html:0.7.2 (transitive dependency of current Detekt version)
```

Comment:
This repo used kotlinx-html directly from the build-verdict plugin and transitively from detekt

Reply:
There is no problem with our build-verdict plugin because it uses correct version of kotlin-html which is available in mavenCentral.